src/LinearLoader/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { theme } from '../theme.js'
+
+const Progress = ({ amount }) => {
+    return (
+        <div>
+            <style jsx>{`
+                div {
+                    width: ${amount}%;
+                }
+            `}</style>
+            <style jsx>{`
+                div {
+                    background-color: ${theme.primary600};
+                    transition: width 0.3s linear;
+                    height: 4px;
+                }
+
+                .determinate {
+                }
+            `}</style>
+        </div>
+    )
+}
+
+const LinearLoader = ({ amount, width, margin, className }) => {
```

Comment:
Renamed to match the terms in https://github.com/dhis2/design-system/blob/master/atoms/loading.md

#237